### ISSUE TITLE
Cannot summarize columns from joined table based on a Saved Question

### ISSUE THREAD

**Issue by flamber** [metabase/metabase#14766]
**Describe the bug**
Cannot summarize columns from joined table based on a Saved Question.
Worked in 0.37.8, but not in 0.38.0-rc4/5 or commit `f5efa3b`

**To Reproduce**
1. Custom question > Sample Dataset > Orders and join Products, and save as "Q1"
![image](https://user-images.githubusercontent.com/1447303/107670383-f6687d00-6c92-11eb-97fb-a00d1f29e04a.png)
2. Custom question > Saved Questions > Q1
3. Summarize Count by Product.Category
![image](https://user-images.githubusercontent.com/1447303/107670737-54956000-6c93-11eb-9584-f10c995f6da1.png)
4. Fails with `Column "source.CATEGORY" not found`

<details><summary>Full stacktrace</summary>

```
2021-02-11 18:03:48,490 INFO api.dataset :: Source query for this query is Card 397
2021-02-11 18:03:48,505 INFO middleware.fetch-source-query :: Fetched source query from Card 397: 
 {:source-table 11, :joins [{:fields :all, :source-table 10, :condition [:= [:field-id 82] [:joined-field "Products" [:field-id 105]]], :alias "Products"}]}

2021-02-11 18:03:48,519 INFO middleware.fetch-source-query :: Fetched source query from Card 397: 
 {:source-table 11, :joins [{:fields :all, :source-table 10, :condition [:= [:field-id 82] [:joined-field "Products" [:field-id 105]]], :alias "Products"}]}

2021-02-11 18:03:49,558 ERROR middleware.catch-exceptions :: Error processing query: null
{:database_id 4,
 :started_at #t "2021-02-11T18:03:48.496588+01:00[Europe/Copenhagen]",
 :state "42S22",
 :json_query
 {:database 4,
  :query {:source-table "card__397", :aggregation [["count"]], :breakout [["field-literal" "CATEGORY" "type/Text"]]},
  :type "query",
  :parameters [],
  :middleware {:js-int-to-string? true, :add-default-userland-constraints? true}},
 :native
 {:query
  "SELECT \"source\".\"CATEGORY\" AS \"CATEGORY\", count(*) AS \"count\" FROM (SELECT \"PUBLIC\".\"ORDERS\".\"ID\" AS \"ID\", \"PUBLIC\".\"ORDERS\".\"USER_ID\" AS \"USER_ID\", \"PUBLIC\".\"ORDERS\".\"PRODUCT_ID\" AS \"PRODUCT_ID\", \"PUBLIC\".\"ORDERS\".\"SUBTOTAL\" AS \"SUBTOTAL\", \"PUBLIC\".\"ORDERS\".\"TAX\" AS \"TAX\", \"PUBLIC\".\"ORDERS\".\"TOTAL\" AS \"TOTAL\", \"PUBLIC\".\"ORDERS\".\"DISCOUNT\" AS \"DISCOUNT\", \"PUBLIC\".\"ORDERS\".\"CREATED_AT\" AS \"CREATED_AT\", \"PUBLIC\".\"ORDERS\".\"QUANTITY\" AS \"QUANTITY\", \"Products\".\"ID\" AS \"Products__ID\", \"Products\".\"EAN\" AS \"Products__EAN\", \"Products\".\"TITLE\" AS \"Products__TITLE\", \"Products\".\"CATEGORY\" AS \"Products__CATEGORY\", \"Products\".\"VENDOR\" AS \"Products__VENDOR\", \"Products\".\"PRICE\" AS \"Products__PRICE\", \"Products\".\"RATING\" AS \"Products__RATING\", \"Products\".\"CREATED_AT\" AS \"Products__CREATED_AT\" FROM \"PUBLIC\".\"ORDERS\" LEFT JOIN \"PUBLIC\".\"PRODUCTS\" \"Products\" ON \"PUBLIC\".\"ORDERS\".\"PRODUCT_ID\" = \"Products\".\"ID\") \"source\" GROUP BY \"source\".\"CATEGORY\" ORDER BY \"source\".\"CATEGORY\" ASC",
  :params nil},
 :status :failed,
 :class org.h2.jdbc.JdbcSQLException,
 :stacktrace
 ["org.h2.message.DbException.getJdbcSQLException(DbException.java:357)"
  "org.h2.message.DbException.get(DbException.java:179)"
  "org.h2.message.DbException.get(DbException.java:155)"
  "org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:150)"
  "org.h2.expression.Alias.optimize(Alias.java:51)"
  "org.h2.command.dml.Select.prepare(Select.java:858)"
  "org.h2.command.Parser.prepareCommand(Parser.java:283)"
  "org.h2.engine.Session.prepareLocal(Session.java:611)"
  "org.h2.engine.Session.prepareCommand(Session.java:549)"
  "org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)"
  "org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:76)"
  "org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:1135)"
  "com.mchange.v2.c3p0.impl.NewProxyConnection.prepareStatement(NewProxyConnection.java:267)"
  "--> driver.sql_jdbc.execute$fn__77359.invokeStatic(execute.clj:236)"
  "driver.sql_jdbc.execute$fn__77359.invoke(execute.clj:234)"
  "driver.sql_jdbc.execute$prepared_statement_STAR_.invokeStatic(execute.clj:254)"
  "driver.sql_jdbc.execute$prepared_statement_STAR_.invoke(execute.clj:251)"
  "driver.sql_jdbc.execute$execute_reducible_query.invokeStatic(execute.clj:387)"
  "driver.sql_jdbc.execute$execute_reducible_query.invoke(execute.clj:374)"
  "driver.sql_jdbc.execute$execute_reducible_query.invokeStatic(execute.clj:383)"
  "driver.sql_jdbc.execute$execute_reducible_query.invoke(execute.clj:374)"
  "driver.sql_jdbc$fn__78943.invokeStatic(sql_jdbc.clj:54)"
  "driver.sql_jdbc$fn__78943.invoke(sql_jdbc.clj:52)"
  "driver.h2$fn__77749.invokeStatic(h2.clj:81)"
  "driver.h2$fn__77749.invoke(h2.clj:78)"
  "query_processor.context$executef.invokeStatic(context.clj:59)"
  "query_processor.context$executef.invoke(context.clj:48)"
  "query_processor.context.default$default_runf.invokeStatic(default.clj:68)"
  "query_processor.context.default$default_runf.invoke(default.clj:66)"
  "query_processor.context$runf.invokeStatic(context.clj:45)"
  "query_processor.context$runf.invoke(context.clj:39)"
  "query_processor.reducible$pivot.invokeStatic(reducible.clj:34)"
  "query_processor.reducible$pivot.invoke(reducible.clj:31)"
  "query_processor.middleware.mbql_to_native$mbql__GT_native$fn__46371.invoke(mbql_to_native.clj:25)"
  "query_processor.middleware.check_features$check_features$fn__45647.invoke(check_features.clj:41)"
  "query_processor.middleware.limit$limit$fn__46357.invoke(limit.clj:37)"
  "query_processor.middleware.cache$maybe_return_cached_results$fn__45298.invoke(cache.clj:211)"
  "query_processor.middleware.optimize_datetime_filters$optimize_datetime_filters$fn__46536.invoke(optimize_datetime_filters.clj:133)"
  "query_processor.middleware.auto_parse_filter_values$auto_parse_filter_values$fn__44445.invoke(auto_parse_filter_values.clj:43)"
  "query_processor.middleware.wrap_value_literals$wrap_value_literals$fn__40802.invoke(wrap_value_literals.clj:147)"
  "query_processor.middleware.annotate$add_column_info$fn__40665.invoke(annotate.clj:579)"
  "query_processor.middleware.permissions$check_query_permissions$fn__45522.invoke(permissions.clj:69)"
  "query_processor.middleware.pre_alias_aggregations$pre_alias_aggregations$fn__47055.invoke(pre_alias_aggregations.clj:40)"
  "query_processor.middleware.cumulative_aggregations$handle_cumulative_aggregations$fn__45720.invoke(cumulative_aggregations.clj:60)"
  "query_processor.middleware.resolve_joined_fields$resolve_joined_fields$fn__47312.invoke(resolve_joined_fields.clj:82)"
  "query_processor.middleware.resolve_joins$resolve_joins$fn__47649.invoke(resolve_joins.clj:207)"
  "query_processor.middleware.add_implicit_joins$add_implicit_joins$fn__44043.invoke(add_implicit_joins.clj:250)"
  "query_processor.middleware.large_int_id$convert_id_to_string$fn__46332.invoke(large_int_id.clj:44)"
  "query_processor.middleware.format_rows$format_rows$fn__46312.invoke(format_rows.clj:74)"
  "query_processor.middleware.desugar$desugar$fn__45786.invoke(desugar.clj:21)"
  "query_processor.middleware.binning$update_binning_strategy$fn__44804.invoke(binning.clj:225)"
  "query_processor.middleware.resolve_fields$resolve_fields$fn__45322.invoke(resolve_fields.clj:24)"
  "query_processor.middleware.add_dimension_projections$add_remapping$fn__43591.invoke(add_dimension_projections.clj:315)"
  "query_processor.middleware.add_implicit_clauses$add_implicit_clauses$fn__43794.invoke(add_implicit_clauses.clj:138)"
  "query_processor.middleware.add_source_metadata$add_source_metadata_for_source_queries$fn__44192.invoke(add_source_metadata.clj:103)"
  "query_processor.middleware.reconcile_breakout_and_order_by_bucketing$reconcile_breakout_and_order_by_bucketing$fn__47252.invoke(reconcile_breakout_and_order_by_bucketing.clj:97)"
  "query_processor.middleware.auto_bucket_datetimes$auto_bucket_datetimes$fn__44392.invoke(auto_bucket_datetimes.clj:139)"
  "query_processor.middleware.resolve_source_table$resolve_source_tables$fn__45369.invoke(resolve_source_table.clj:45)"
  "query_processor.middleware.parameters$substitute_parameters$fn__47037.invoke(parameters.clj:111)"
  "query_processor.middleware.resolve_referenced$resolve_referenced_card_resources$fn__45421.invoke(resolve_referenced.clj:79)"
  "query_processor.middleware.expand_macros$expand_macros$fn__46042.invoke(expand_macros.clj:155)"
  "query_processor.middleware.add_timezone_info$add_timezone_info$fn__44201.invoke(add_timezone_info.clj:15)"
  "query_processor.middleware.splice_params_in_response$splice_params_in_response$fn__48015.invoke(splice_params_in_response.clj:32)"
  "query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__47263$fn__47267.invoke(resolve_database_and_driver.clj:31)"
  "driver$do_with_driver.invokeStatic(driver.clj:60)"
  "driver$do_with_driver.invoke(driver.clj:56)"
  "query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__47263.invoke(resolve_database_and_driver.clj:25)"
  "query_processor.middleware.fetch_source_query$resolve_card_id_source_tables$fn__46260.invoke(fetch_source_query.clj:264)"
  "query_processor.middleware.store$initialize_store$fn__48024$fn__48025.invoke(store.clj:11)"
  "query_processor.store$do_with_store.invokeStatic(store.clj:44)"
  "query_processor.store$do_with_store.invoke(store.clj:38)"
  "query_processor.middleware.store$initialize_store$fn__48024.invoke(store.clj:10)"
  "query_processor.middleware.validate$validate_query$fn__48033.invoke(validate.clj:10)"
  "query_processor.middleware.normalize_query$normalize$fn__46384.invoke(normalize_query.clj:22)"
  "query_processor.middleware.add_rows_truncated$add_rows_truncated$fn__44061.invoke(add_rows_truncated.clj:35)"
  "query_processor.middleware.results_metadata$record_and_return_metadata_BANG_$fn__48000.invoke(results_metadata.clj:147)"
  "query_processor.middleware.constraints$add_default_userland_constraints$fn__45663.invoke(constraints.clj:42)"
  "query_processor.middleware.process_userland_query$process_userland_query$fn__47126.invoke(process_userland_query.clj:135)"
  "query_processor.middleware.catch_exceptions$catch_exceptions$fn__45606.invoke(catch_exceptions.clj:173)"
  "query_processor.reducible$async_qp$qp_STAR___37444$thunk__37445.invoke(reducible.clj:103)"
  "query_processor.reducible$async_qp$qp_STAR___37444.invoke(reducible.clj:109)"
  "query_processor.reducible$sync_qp$qp_STAR___37453$fn__37456.invoke(reducible.clj:135)"
  "query_processor.reducible$sync_qp$qp_STAR___37453.invoke(reducible.clj:134)"
  "query_processor$process_userland_query.invokeStatic(query_processor.clj:235)"
  "query_processor$process_userland_query.doInvoke(query_processor.clj:231)"
  "query_processor$fn__48079$process_query_and_save_execution_BANG___48088$fn__48091.invoke(query_processor.clj:247)"
  "query_processor$fn__48079$process_query_and_save_execution_BANG___48088.invoke(query_processor.clj:239)"
  "query_processor$fn__48123$process_query_and_save_with_max_results_constraints_BANG___48132$fn__48135.invoke(query_processor.clj:259)"
  "query_processor$fn__48123$process_query_and_save_with_max_results_constraints_BANG___48132.invoke(query_processor.clj:252)"
  "api.dataset$fn__54233$fn__54236.invoke(dataset.clj:55)"
  "query_processor.streaming$streaming_response_STAR_$fn__54214$fn__54215.invoke(streaming.clj:72)"
  "query_processor.streaming$streaming_response_STAR_$fn__54214.invoke(streaming.clj:71)"
  "async.streaming_response$do_f_STAR_.invokeStatic(streaming_response.clj:65)"
  "async.streaming_response$do_f_STAR_.invoke(streaming_response.clj:63)"
  "async.streaming_response$do_f_async$fn__16055.invoke(streaming_response.clj:84)"],
 :context :ad-hoc,
 :error
 "Column \"source.CATEGORY\" not found; SQL statement:\n-- Metabase:: userID: 1 queryType: MBQL queryHash: 2cc21f69c16526fa75f83ecbc9b9e7a8cff7ae52ee05b7d036d85bf8c9e06e57\nSELECT \"source\".\"CATEGORY\" AS \"CATEGORY\", count(*) AS \"count\" FROM (SELECT \"PUBLIC\".\"ORDERS\".\"ID\" AS \"ID\", \"PUBLIC\".\"ORDERS\".\"USER_ID\" AS \"USER_ID\", \"PUBLIC\".\"ORDERS\".\"PRODUCT_ID\" AS \"PRODUCT_ID\", \"PUBLIC\".\"ORDERS\".\"SUBTOTAL\" AS \"SUBTOTAL\", \"PUBLIC\".\"ORDERS\".\"TAX\" AS \"TAX\", \"PUBLIC\".\"ORDERS\".\"TOTAL\" AS \"TOTAL\", \"PUBLIC\".\"ORDERS\".\"DISCOUNT\" AS \"DISCOUNT\", \"PUBLIC\".\"ORDERS\".\"CREATED_AT\" AS \"CREATED_AT\", \"PUBLIC\".\"ORDERS\".\"QUANTITY\" AS \"QUANTITY\", \"Products\".\"ID\" AS \"Products__ID\", \"Products\".\"EAN\" AS \"Products__EAN\", \"Products\".\"TITLE\" AS \"Products__TITLE\", \"Products\".\"CATEGORY\" AS \"Products__CATEGORY\", \"Products\".\"VENDOR\" AS \"Products__VENDOR\", \"Products\".\"PRICE\" AS \"Products__PRICE\", \"Products\".\"RATING\" AS \"Products__RATING\", \"Products\".\"CREATED_AT\" AS \"Products__CREATED_AT\" FROM \"PUBLIC\".\"ORDERS\" LEFT JOIN \"PUBLIC\".\"PRODUCTS\" \"Products\" ON \"PUBLIC\".\"ORDERS\".\"PRODUCT_ID\" = \"Products\".\"ID\") \"source\" GROUP BY \"source\".\"CATEGORY\" ORDER BY \"source\".\"CATEGORY\" ASC [42122-197]",
 :row_count 0,
 :running_time 0,
 :preprocessed
 {:database 4,
  :query
  {:aggregation [[:aggregation-options [:count] {:name "count"}]],
   :breakout [[:field-literal "CATEGORY" :type/Text]],
   :source-query
   {:source-table 11,
    :fields
    [[:field-id 83]
     [:field-id 80]
     [:field-id 82]
     [:field-id 84]
     [:field-id 87]
     [:field-id 88]
     [:field-id 81]
     [:datetime-field [:field-id 86] :default]
     [:field-id 85]
     [:joined-field "Products" [:field-id 105]]
     [:joined-field "Products" [:field-id 102]]
     [:joined-field "Products" [:field-id 106]]
     [:joined-field "Products" [:field-id 107]]
     [:joined-field "Products" [:field-id 109]]
     [:joined-field "Products" [:field-id 104]]
     [:joined-field "Products" [:field-id 103]]
     [:datetime-field [:joined-field "Products" [:field-id 108]] :default]],
    :joins [{:strategy :left-join, :source-table 10, :condition [:= [:field-id 82] [:joined-field "Products" [:field-id 105]]], :alias "Products"}]},
   :source-metadata
   [{:name "ID",
     :display_name "ID",
     :base_type :type/BigInteger,
     :special_type :type/PK,
     :id 83,
     :field_ref [:field-id 83],
     :description "This is a unique ID for the product. It is also called the “Invoice number” or “Confirmation number” in customer facing emails and screens.",
     :fingerprint nil}
    {:name "USER_ID",
     :display_name "User ID",
     :base_type :type/Integer,
     :special_type :type/FK,
     :id 80,
     :field_ref [:field-id 80],
     :description
     "The id of the user who made this order. Note that in some cases where an order was created on behalf of a customer who phoned the order in, this might be the employee who handled the request.",
     :fingerprint {:global {:distinct-count 929, :nil% 0.0}}}
    {:name "PRODUCT_ID",
     :display_name "Product ID",
     :base_type :type/Integer,
     :special_type :type/FK,
     :id 82,
     :field_ref [:field-id 82],
     :description "The product ID. This is an internal identifier for the product, NOT the SKU.",
     :fingerprint {:global {:distinct-count 200, :nil% 0.0}}}
    {:name "SUBTOTAL",
     :display_name "Subtotal",
     :base_type :type/Float,
     :special_type :type/Currency,
     :id 84,
     :field_ref [:field-id 84],
     :description "The raw, pre-tax cost of the order. Note that this might be different in the future from the product price due to promotions, credits, etc.",
     :fingerprint
     {:global {:distinct-count 340, :nil% 0.0},
      :type {:type/Number {:min 15.691943673970439, :max 148.22900526552291, :avg 77.01295465356547, :sd 32.53705013056317, :q1 49.74894519060184, :q3 105.42965746993103}}}}
    {:name "TAX",
     :display_name "Tax",
     :base_type :type/Float,
     :special_type :type/Currency,
     :id 87,
     :field_ref [:field-id 87],
     :description
     "This is the amount of local and federal taxes that are collected on the purchase. Note that other governmental fees on some products are not included here, but instead are accounted for in the subtotal.",
     :fingerprint
     {:global {:distinct-count 797, :nil% 0.0}, :type {:type/Number {:min 0.0, :max 11.12, :avg 3.8722100000000004, :sd 2.3206651358900316, :q1 2.273340386603857, :q3 5.337275338216307}}}}
    {:name "TOTAL",
     :display_name "Total",
     :base_type :type/Float,
     :special_type :type/Income,
     :id 88,
     :field_ref [:field-id 88],
     :description "The total billed amount.",
     :fingerprint
     {:global {:distinct-count 10000, :nil% 0.0},
      :type {:type/Number {:min 12.061602936923117, :max 238.32732001721533, :avg 82.96014815230805, :sd 38.35967664847571, :q1 52.006147617878135, :q3 109.55803018499738}}}}
    {:name "DISCOUNT",
     :display_name "Discount",
     :base_type :type/Float,
     :special_type :type/Discount,
     :id 81,
     :field_ref [:field-id 81],
     :description "Discount amount.",
     :fingerprint
     {:global {:distinct-count 701, :nil% 0.898},
      :type {:type/Number {:min 0.17088996672584322, :max 61.69684269960571, :avg 5.161255547580326, :sd 3.053663125001991, :q1 2.9786226681458743, :q3 7.338187788658235}}}}
    {:description "The date and time an order was submitted.",
     :special_type :type/CreationTimestamp,
     :unit :default,
     :name "CREATED_AT",
     :field_ref [:datetime-field [:field-id 86] :default],
     :id 86,
     :display_name "Created At",
     :fingerprint {:global {:distinct-count 10001, :nil% 0.0}, :type {:type/DateTime {:earliest "2016-05-01T00:56:13.352Z", :latest "2020-04-19T20:07:15.657Z"}}},
     :base_type :type/DateTime}
    {:name "QUANTITY",
     :display_name "Quantity",
     :base_type :type/Integer,
     :special_type :type/Quantity,
     :id 85,
     :field_ref [:field-id 85],
     :description "Number of products bought.",
     :fingerprint {:global {:distinct-count 62, :nil% 0.0}, :type {:type/Number {:min 0.0, :max 100.0, :avg 3.7015, :sd 4.214258386403798, :q1 1.755882607764982, :q3 4.882654507928044}}}}
    {:name "ID_2",
     :display_name "Products → ID",
     :base_type :type/BigInteger,
     :special_type :type/PK,
     :id 105,
     :field_ref [:joined-field "Products" [:field-id 105]],
     :description "The numerical product number. Only used internally. All external communication should use the title or EAN.",
     :fingerprint nil}
    {:name "EAN",
     :display_name "Products → Ean",
     :base_type :type/Text,
     :special_type :type/Title,
     :id 102,
     :field_ref [:joined-field "Products" [:field-id 102]],
     :description "The international article number. A 13 digit number uniquely identifying the product.",
     :fingerprint {:global {:distinct-count 200, :nil% 0.0}, :type {:type/Text {:percent-json 0.0, :percent-url 0.0, :percent-email 0.0, :percent-state 0.0, :average-length 13.0}}}}
    {:name "TITLE",
     :display_name "Products → Title",
     :base_type :type/Text,
     :special_type :type/Name,
     :id 106,
     :field_ref [:joined-field "Products" [:field-id 106]],
     :description "The name of the product as it should be displayed to customers.",
     :fingerprint {:global {:distinct-count 199, :nil% 0.0}, :type {:type/Text {:percent-json 0.0, :percent-url 0.0, :percent-email 0.0, :percent-state 0.0, :average-length 21.495}}}}
    {:name "CATEGORY",
     :display_name "Products → Category",
     :base_type :type/Text,
     :special_type :type/Category,
     :id 107,
     :field_ref [:joined-field "Products" [:field-id 107]],
     :description "The type of product, valid values include: Doohicky, Gadget, Gizmo and Widget",
     :fingerprint {:global {:distinct-count 4, :nil% 0.0}, :type {:type/Text {:percent-json 0.0, :percent-url 0.0, :percent-email 0.0, :percent-state 0.0, :average-length 6.375}}}}
    {:name "VENDOR",
     :display_name "Products → Vendor",
     :base_type :type/Text,
     :special_type :type/Company,
     :id 109,
     :field_ref [:joined-field "Products" [:field-id 109]],
     :description "The source of the product.",
     :fingerprint {:global {:distinct-count 200, :nil% 0.0}, :type {:type/Text {:percent-json 0.0, :percent-url 0.0, :percent-email 0.0, :percent-state 0.0, :average-length 20.6}}}}
    {:name "PRICE",
     :display_name "Products → Price",
     :base_type :type/Float,
     :special_type nil,
     :id 104,
     :field_ref [:joined-field "Products" [:field-id 104]],
     :description "The list price of the product. Note that this is not always the price the product sold for due to discounts, promotions, etc.",
     :fingerprint
     {:global {:distinct-count 170, :nil% 0.0},
      :type {:type/Number {:min 15.691943673970439, :max 98.81933684368194, :avg 55.74639966792074, :sd 21.711481557852057, :q1 37.25154462926434, :q3 75.45898071609447}}}}
    {:name "RATING",
     :display_name "Products → Rating",
     :base_type :type/Float,
     :special_type :type/Score,
     :id 103,
     :field_ref [:joined-field "Products" [:field-id 103]],
     :description "The average rating users have given the product. This ranges from 1 - 5",
     :fingerprint {:global {:distinct-count 23, :nil% 0.0}, :type {:type/Number {:min 0.0, :max 5.0, :avg 3.4715, :sd 1.3605488657451452, :q1 3.5120465053408525, :q3 4.216124969497314}}}}
    {:description "The date the product was added to our catalog.",
     :special_type :type/CreationTimestamp,
     :unit :default,
     :name "CREATED_AT_2",
     :field_ref [:datetime-field [:joined-field "Products" [:field-id 108]] :default],
     :id 108,
     :display_name "Products → Created At",
     :fingerprint {:global {:distinct-count 200, :nil% 0.0}, :type {:type/DateTime {:earliest "2016-04-27T01:29:55.147Z", :latest "2019-04-15T19:34:19.931Z"}}},
     :base_type :type/DateTime}],
   :order-by [[:asc [:field-literal "CATEGORY" :type/Text]]]},
  :type :query,
  :middleware {:js-int-to-string? true, :add-default-userland-constraints? true},
  :info
  {:executed-by 1,
   :context :ad-hoc,
   :card-id 397,
   :nested? true,
   :query-hash [44, -62, 31, 105, -63, 101, 38, -6, 117, -8, 62, -53, -55, -71, -25, -88, -49, -9, -82, 82, -18, 5, -73, -48, 54, -40, 91, -8, -55, -32, 110, 87]},
  :constraints {:max-results 10000, :max-results-bare-rows 2000}},
 :data {:rows [], :cols []}}
```

</details>

**Information about your Metabase Installation:**
Worked in 0.37.8, but not in 0.38.0-rc4/5 or commit `f5efa3b`

**Additional context**
Only tested without FK remapping because of 14745 would generate an error in earlier step.

**Comment by camsaul** [comment]
Should be fixed by #14745

**Comment by flamber** [comment]
I have tested latest commit `f5efa3b` and that fails too

**Comment by camsaul** [comment]
Sorry @flamber , I meant once I finished #14745and fixed the regression

**Comment by camsaul** [comment]
Fixed by #14745
